org.eclipse.scanning.test/src/org/eclipse/scanning/test/validation/ScanRequestValidationTest.java

```diff
@@ -40,7 +40,7 @@ public void nullDetectorModelsAllowed() throws Exception {
 
 		GridModel gmodel = new GridModel("stage_x", "stage_y");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "stage_y" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "stage_x" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)